### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Shevtsooov**
[DEMO LINK](https://Shevtsooov.github.io/react_person/)

### CODE REVIEW

File: src/components/Person/Person.jsx [hunk 1]
```diff
@@ -1 +1,31 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import './Person.css';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
```
Comment (dinaminahi) [inline src/components/Person/Person.jsx:5]:
```suggestion
  const { 
      name, 
      age, 
      sex, 
      isMarried, 
      partnerName 
   } = person;
```

**Review by dinaminahi — CHANGES_REQUESTED** [review]
Good job 👍
One moment with readability

**Comment by Shevtsooov** [comment]
@dinaminahi Hi, thanks for the review. The changes have been applied.
Please let me know if this is the proper solution.

File: src/components/Person/Person.jsx [hunk 2]
```diff
@@ -1 +1,40 @@
...
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  let partnerRole = 'wife';
+
+  if (sex === 'f') {
+    partnerRole = 'husband';
+  }
```
Comment (silchyn) [inline src/components/Person/Person.jsx:17]:
```suggestion
  let partnerRole = sex === 'f'
    ? 'husband'
    : 'wife';
```

**Review by silchyn — APPROVED** [review]

**Pull request closed without merging**